tests/ui/parser/issues/issue-111416.stderr

```diff
@@ -1,18 +1,33 @@
-error: invalid `struct` delimiters or `fn` call arguments
-  --> $DIR/issue-111416.rs:2:14
+error: expected identifier, found `:`
+  --> $DIR/issue-111416.rs:2:30
    |
 LL |     let my = monad_bind(mx, T: Try);
-   |              ^^^^^^^^^^^^^^^^^^^^^^
-   |
-help: if `monad_bind` is a struct, use braces as delimiters
+   |                              ^ expected identifier
    |
-LL |     let my = monad_bind { mx, T: Try };
-   |                         ~            ~
-help: if `monad_bind` is a function, use the arguments directly
+help: if this is a parameter, remove the name for the parameter
    |
 LL -     let my = monad_bind(mx, T: Try);
 LL +     let my = monad_bind(mx, Try);
    |
 
-error: aborting due to 1 previous error
+error[E0425]: cannot find value `mx` in this scope
+  --> $DIR/issue-111416.rs:2:25
+   |
+LL |     let my = monad_bind(mx, T: Try);
+   |                         ^^ not found in this scope
+
+error[E0425]: cannot find value `Try` in this scope
+  --> $DIR/issue-111416.rs:2:32
+   |
+LL |     let my = monad_bind(mx, T: Try);
+   |                                ^^^ not found in this scope
+
+error[E0425]: cannot find function `monad_bind` in this scope
+  --> $DIR/issue-111416.rs:2:14
+   |
+LL |     let my = monad_bind(mx, T: Try);
+   |              ^^^^^^^^^^ not found in this scope
```

Comment:
These errors are now correct. You need to change https://github.com/rust-lang/rust/blob/359bd1d7f77bd1ffea3e403d0679b3de9143c08f/tests/ui/parser/issues/issue-111416.rs to be 

```rust
fn main() {
    let my = monad_bind(mx, T: Try);
    //~^ ERROR invalid `struct` delimiters or `fn` call arguments
    //~| ERROR cannot find value `mx`
    //~| ERROR cannot find value `Try`
    //~| ERROR cannot find function `monad_bind`
}
```

Reply:
Changes made.